sentry-ruby/lib/sentry/utils/logging_helper.rb

```diff
@@ -2,21 +2,23 @@
 
 module Sentry
   module LoggingHelper
+    attr_reader :logger
```

Comment:
Maybe we should name this `sdk_logger` too to standardize how we access internal logging. It'll make grepping the codebase easier.